pkg/testcase/upgradenodereplacement.go

```diff
@@ -45,7 +45,7 @@ func TestUpgradeReplaceNode(cluster *shared.Cluster, version string) {
 	Expect(scpErr).NotTo(HaveOccurred(), scpErr)
 	shared.LogLevel("info", "Scp files to new server nodes done\n")
 
-	serverLeaderIP := cluster.ServerIPs[0]
+	serverLeaderIP := newExternalServerIps[0]
```

Comment:
@aganesh-suse sorry please revert this change, will revisit this

Reply:
yes please

Reply:
i suspect that it is talking to the old IP to get the token to copy on to the new set of nodes maybe. so i dont think we should be using the new IP for this line. reverting the change.

Reply:
https://github.com/rancher/distros-test-framework/pull/149#discussion_r1723988444